src/main/java/org/assertj/core/internal/Files.java

```diff
@@ -480,6 +482,13 @@ public void assertIsDirectoryContaining(AssertionInfo info, File actual, String
     assertIsDirectoryContaining(info, actual, pathMatcher, format("the '%s' pattern", syntaxAndPattern));
   }
 
+  public void assertIsDirectoryRecursivelyContaining(AssertionInfo info, File actual, String syntaxAndPattern) {
+    requireNonNull(syntaxAndPattern, "The syntax and pattern to build PathMatcher should not be null");
+    Predicate<File> pathMatcher = pathMatcher(info, actual, syntaxAndPattern);
```

Comment:
rename to `fileMatcher` as it is a File predicate

Reply:
Not wrong but that is consistent with the invoked method name. To make the whole thing consistent, we should also rename the method : `pathMatcher() `to `fileMatcher()` and also to do the same rename refactoring for other `pathMatcher()` callers and variables assigned to. Agreed for that name refactoring ?

Reply:
yes, thank you!

Reply:
done